flask/notejam/models.py

```diff
@@ -6,11 +6,12 @@
 
 from notejam import db
 
+PASSWORD_LEN = len(generate_password_hash('foo'))
 
 class User(db.Model, UserMixin):
     id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(120), unique=True)
-    password = db.Column(db.String(36))
+    password = db.Column(db.String(PASSWORD_LEN))
```

Comment:
Ok, I'll do it soon :)
On Sep 28, 2015 1:28 PM, "Serhii Komar" <notifications@github.com> wrote:

> In flask/notejam/models.py
> <https://github.com/komarserjio/notejam/pull/47#discussion_r40537692>:
>
> >
> >  class User(db.Model, UserMixin):
> >      id = db.Column(db.Integer, primary_key=True)
> >      email = db.Column(db.String(120), unique=True)
> > -    password = db.Column(db.String(36))
> > +    password = db.Column(db.String(PASSWORD_LEN))
>
> And that day we can update the column :) It's just a learning app, let's
> keep it as simple as possible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/komarserjio/notejam/pull/47/files#r40537692>.
>